android/.classpath

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
You probably don't want to include eclipse files in your commit.

Reply:
Yeah, this issue occures very rare with specific devices in our customers. 
Such as 57513  OPPO  CPH1853  android  8.1.0, samsung  SM-T295  android  9, OPPO  CPH1911  android  9, samsung  SM-A750GN  android  9, OnePlus  ONEPLUS A5010  android  9, LG K10 android 5.1.1 and also emulators whichs android below android 5.1.1